x-pack/legacy/plugins/reporting/server/routes/lib/get_document_payload.ts

```diff
@@ -33,20 +38,22 @@ const getReportingHeaders = (output: any, exportType: any) => {
   return metaDataHeaders;
 };
 
-function getDocumentPayloadFn(server: any) {
-  const exportTypesRegistry = server.plugins.reporting.exportTypesRegistry;
+function getDocumentPayloadFn(server: ServerFacade) {
+  const exportTypesRegistry = server.plugins.reporting!.exportTypesRegistry;
 
-  function encodeContent(content: string, exportType: any) {
+  function encodeContent(content: string | null, exportType: ExportTypeDefinition) {
     switch (exportType.jobContentEncoding) {
       case 'base64':
-        return Buffer.from(content, 'base64');
+        return content ? Buffer.from(content, 'base64') : content;
```

Comment:
This is an interesting change, could you explain why we only do `Buffer.from` when it's defined (assuming the truthy check is for that)

Reply:
It is because `Buffer.from` complains about `null`, so this may have been a bug hiding here.